src/components/dashboard/Dashboard.js

```diff
@@ -84,6 +84,7 @@ export type DashboardProps = {
 }
 const Dashboard = props => {
   const { screenProps, styles, theme }: DashboardProps = props
+  const [getNextFeedAllowed, setGetNextFeedAllowed] = useState(true)
```

Comment:
first element of the array returned by useState is an **state value**, not an state **getter** (e.g. function that returns the value).

so it shouldn't have 'get' prefix. Namings in React Hooks is very important thing!

correctly should be:
```
const [isNextFeedAllowed, setNextFeedAllowed] = useState(true);
```
or
```
const [isNextFeedAllowed, setIsNextFeedAllowed] = useState(true);
```
(both semantically are right but first one looks better).

if variable isn't boolean - of course we don;t using 'is' prefix

```
const [count, setCount] = useState(0);
```